common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/PipelineStorageStream.scala

```diff
@@ -133,12 +134,37 @@ object PipelineStorageStream extends Logging {
     indexer: Indexer[T])(implicit
                          ec: ExecutionContext,
                          indexable: Indexable[T]) =
-    Flow[(Message, Option[T])]
-      .collect { case (msg, Some(document)) => Bundle(msg, document) }
+    Flow[(Message, List[T])]
+      .collect {
+        case (msg, items @ _ :: _) =>
```

Comment:
This is for filtering out empty items right :thinking: ?

Reply:
Yes, cause empty lists are dealt with in the other branch of the broad cast, the identityFlow